components/ide/jetbrains/image/status/main.go

```diff
@@ -225,6 +225,7 @@ func run(wsInfo *supervisor.WorkspaceInfoResponse) {
 	args = append(args, "run")
 	args = append(args, wsInfo.GetCheckoutLocation())
 	cmd := remoteDevServerCmd(args)
+	cmd.Env = append(cmd.Env, "JETBRAINS_GITPOD_WORKSPACE_HOST="+wsInfo.GetWorkspaceUrl())
```

Comment:
it should be host, not url

Reply:
Ah, you mean to use this one?

https://github.com/gitpod-io/gitpod/blob/79542d659c20c7be71d912e458798577b4d5f732/components/supervisor-api/info.proto#L56

Or to create a new property?

Reply:
yes, you can do it like that or just parse url and get host part to be sure

Reply:
Code updated!
I"ve also updated the PR description.